pkg/jwt/verification.go

```diff
@@ -1,29 +0,0 @@
-package jwt
-
-import (
-	"context"
-
-	"github.com/pkg/errors"
-	didint "github.com/tbd54566975/ssi-service/internal/did"
-	"github.com/tbd54566975/ssi-service/internal/keyaccess"
-	"github.com/tbd54566975/ssi-service/internal/util"
-	"github.com/tbd54566975/ssi-service/pkg/service/did/resolve"
-)
-
-// VerifyTokenFromDID verifies that the information in the token was digitally signed by the public key associated with
-// the public key of the verification method of the did's document. The passed in resolver is used to map from the did
-// to the did document.
-func VerifyTokenFromDID(ctx context.Context, did string, token keyaccess.JWT, s resolve.Resolver) error {
```

Comment:
I had first put it in internal since it's not tied to any service, but then I saw the dependency on resolver, which is a part of the DID service

then it makes sense that if there's a DID service dependency it shouldn't be in internal...since I had imagined internal only depending on itself

Reply:
disregard, moved back